index.js

```diff
@@ -189,3 +210,37 @@ function addStartupSupportabilities(agent) {
     }
   }
 }
+
+/**
+ * Used to determine how the agent is getting loaded:
+ *  1. -r newrelic
+ *  2. --loader newrelic/esm-loader.mjs
+ *  3. require('newrelic')
+ *
+ *  Then a supportability metric is loaded to decide.
+ *  Note: We already take care of scenario #2 in newrelic/esm-loader.mjs
+ *
+ * @param {Agent} agent active NR agent
+ */
+function recordLoaderMetric(agent) {
+  const dashR = process.execArgv.indexOf('-r')
```

Comment:
You can specify `-r` multiple times on the CLI, like I usually do `node -r dotenv/config -r newrelic file.js`, but `indexOf` only returns the index of the first found one, so I think we have to update this to handle that scenario, otherwise we'll fall into the else if?

Reply:
yes, I rushed this to PR last week and had that on my list and forgot to do. thanks for catching